test/unit/test_ts_environment.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2019-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
You probably want to cover `is_env_set` as well : https://github.com/aws/sagemaker-pytorch-inference-toolkit/blob/master/src/sagemaker_pytorch_serving_container/ts_environment.py#L58-L64

Reply:
Added test for `is_env_set()` in `test_ts_environment.py`.